keps/sig-storage/0000-storage-capacity-scoring-for-dynamic-provisioning/README.md

```diff
@@ -0,0 +1,938 @@
+avoid requiring implementors to split their attention between writing release
+notes and implementing the feature itself. KEP editors and SIG Docs
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+## Motivation
+
+On the following cases, it is needed to schedule pods considering storage capacity.
+
+- We want to select a node with as much free space as possible so that we can resize after a node-local PV is scheduled, and use storage as evenly as possible among the nodes.
+- Or, we want to select a node with less free node space to reduce the number of nodes as much as possible.
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+### Goals
+
+By considering the free space of Dynamic Provisioning, it is possible to schedule according to the free space of the volume.
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+### Non-Goals
+
+- Change the way its score with static and dynamic
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+## Proposal
+
+- Dynamic Provisioning can be scored based on available space
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1
+
+If we want to leave room for volume expansion after node allocation, we can provision with free space in consideration during Dynamic Provisioning, so that nodes with the maximum amount of free space required can be prioritized.
```

Comment:
> we can provision with free space in consideration during Dynamic Provisioning, so that nodes with the maximum amount of free space required can be prioritized.

この辺りの意味が読み取れませんでした。